src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10676,7 +10676,13 @@ private ExpressionSyntax ParseArrayOrObjectCreationExpression()
                         SyntaxFactory.MissingToken(SyntaxKind.CloseParenToken));
                 }
 
-                return _syntaxFactory.ObjectCreationExpression(@new, type, argumentList, initializer);
+                var objectCreation = _syntaxFactory.ObjectCreationExpression(@new, type, argumentList, initializer);
+                if (type.Kind == SyntaxKind.TupleType)
+                {
+                    objectCreation = this.AddError(objectCreation, objectCreation, ErrorCode.ERR_NewWithTupleTypeSyntax, type.ToString());
```

Comment:
It is more common that we do "currentToken.Text" which also can be multiline. 
I think the error will not be nice looking.  Multiline errors are not nice looking in general. 

I do not think there is a lot of choice here other than not specify the type in the error at all.

---
In reply to: [81653476](https://github.com/dotnet/roslyn/pull/14161#discussion_r81653476) [](ancestors = 81653476)